content/docs/user-guide/components/reference/terrain/world.md

```diff
@@ -11,6 +11,17 @@ The **Terrain World** component provides the data required for other terrain sys
 You must add this component to the *Level* entity, the parent of all entities in an **Open 3D Engine (O3DE)** level.
 {{< /important >}}
 
+## Usage 
+
+The **Min Height** and **Max Height** bounds should be set to as small a range as possible for the most accurate height values. For instance, the terrain renderer uses 16 bit height values, which provides 65536 possible distinct heights. If the difference between the min and max height is 1km, that's a resolution of about 1.5cm.
```

Comment:
16 bit -> 16 bits

Reply:
In this case "16 bit" is correct

Reply:
Nit: Actually it should be "16-bit". "1km" and "1.5cm" should also have a space between the number and the unit. ("1 km", "1.5 cm")